frontend/packages/policy-editor/src/components/ExpandablePolicyCard/ExpandablePolicyCard.tsx

```diff
@@ -193,12 +200,18 @@ export const ExpandablePolicyCard = ({
    * resource block is set to the resource's ID and type.
    */
   const handleClickAddResource = () => {
-    const newResource: PolicyRuleResourceType[] = [
-      {
-        type: resourceType,
-        id: resourceId,
-      },
-    ];
+    const newResource: PolicyRuleResourceType[] =
+      usageType === 'app'
+        ? [
+            { type: `${resourceType}:[org]`, id: '[ORG]' },
+            { type: `${resourceType}:[app]`, id: '[APP]' },
+          ]
+        : [
+            {
+              type: resourceType,
+              id: resourceId,
+            },
+          ];
```

Comment:
This code looks quite similar to PolicyEditor line154. I suggest creating utils to make the code more DRY. 😄